### ISSUE TITLE
Textprogramming should have a mode where all tabs a synchronized

### ISSUE THREAD

**Issue by basilstotz** [aseba-community/aseba#534]
Hello,

when programming for several identical nodes (i.e. all nodes have the identical code) it is not easy to manage that the code is really identical for all nodes. Every change in one tab has to be duplicated in all others.

I think it be nice to have a editor mode which keeps the content of all tabs identical.

regards 

Basil

**Comment by stephanemagnenat** [comment]
I agree, the way I see this would be an option in the edit menu, called
"Propagate to similarly-named nodes" that copy the currently-edited code
to other nodes of same name. Do you like this approach?

**Comment by basilstotz** [comment]
Thx, that's exactly what I thought.

**Comment by stephanemagnenat** [comment]
I do not have the engineering resources to work on that right now, but pull requests are welcome :-)
